addons/controllers/clusterbootstrap_controller.go

```diff
@@ -949,7 +1095,7 @@ func (r *ClusterBootstrapReconciler) updateValuesFromSecret(cluster *clusterapiv
 		newSecret = secret.DeepCopy()
 		newSecret.ObjectMeta.Reset()
 		newSecret.Name = fmt.Sprintf("%s-%s-package", cluster.Name, pkgRefName)
-		newSecret.Namespace = bootstrap.Namespace
```

Comment:
Curious why this change is being made. While this works, it is technically namespace of ClusterBootstrap.

Reply:
Because I wanted to minimize the parameters passed into each function. Previuosly, the only purpose of passing in bootstrap is to use its namespace. While the namespace of it will always be the same as cluster.namespace, so I just removed it.
```
clusterBootstrap := &runtanzuv1alpha3.ClusterBootstrap{}
err := r.Client.Get(r.context, client.ObjectKeyFromObject(cluster), clusterBootstrap)
```